internal/controller/user.go

```diff
@@ -0,0 +1,79 @@
+package controller
+
+import (
+	"github.com/avialog/backend/internal/dto"
+	"github.com/avialog/backend/internal/service"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type UserController interface {
+	GetProfile(*gin.Context)
+	UpdateProfile(*gin.Context)
+}
+
+type userController struct {
+	userService service.UserService
+}
+
+func newUserController(userService service.UserService) UserController {
+	return &userController{userService: userService}
+}
+
+func (u *userController) GetProfile(c *gin.Context) {
+	// there will be getting user id from JWT token
```

Comment:
Please add `TODO` like 
```
// TODO add getting user id from JWT token
```

Some IDEs like Intellij scans for that TODO statements and suggest developer to take a glance on it.